WMF Framework/WMFContentGroup+Extensions.m

```diff
@@ -332,7 +332,7 @@ - (void)updateContentPreviewWithContent:(id)content {
         case WMFContentGroupKindNews:
         case WMFContentGroupKindUnknown:
         default: {
-            id<NSCoding> firstObject = contentArray.firstObject ?: content;
+            id<NSSecureCoding> firstObject = contentArray.firstObject ?: content;
```

Comment:
Seems contentArray could contain non-conforming Mantle objects, WMFFeedTopReadArticlePreview for example.

Reply:
Good catch! This should still be NSCoding as the transformer hasn't been changed in the data model yet. I initially went down that path but didn't back out all the changes.